lib/utils/helpers.ts

```diff
@@ -89,10 +89,10 @@ export async function sudo(
 	await executeWithPrivileges(command, stderr, isCLIcmd);
 }
 
-export function runCommand<T>(commandArgs: string[]): Promise<T> {
+export async function runCommand<T>(commandArgs: string[]): Promise<T> {
 	const { isSubcommand } =
 		require('../preparser') as typeof import('../preparser');
-	if (isSubcommand(commandArgs)) {
+	if (await isSubcommand(commandArgs)) {
```

Comment:
This was previously always truthy, the typescript update caught this bug but I wonder if the unconditional behavior might actually be necessary, @pdcastro ?

Reply:
Maybe, needs testing. Currently focusing on CLI v13. cc: @srlowe

Reply:
I checked and it's actually only used in 3 places and they're all sub commands, which explains why this was working before